static/js/controllers/reports.js

```diff
@@ -106,23 +126,38 @@ var _ = require('underscore'),
         $scope.displayFields = fields;
       };
 
+      var setActionBar = function() {
+        var model = {
+          doc: _.pluck($scope.selected, 'report')
```

Comment:
Good call. I've renamed it to "selected" as this is more descriptive than "doc" or "docs".